ethcore/vm/src/schedule.rs

```diff
@@ -151,6 +151,8 @@ pub struct WasmCosts {
 	pub opcodes_div: u32,
 	/// Whether create2 extern function is activated.
 	pub have_create2: bool,
+	/// Does it have a GASLEFT instruction
```

Comment:
I'm not sure why it is referred as a "GASLEFT instruction". I believe it may be quite confusing: GASLEFT is a real concept yet which has nothing to do with wasm which instruction set isn't affect by the KIP6.

Better to borrow wording from the field above.

Reply:
GASLEFT is indeed not an opcode and should not be referred as such